lib/v2/zhubai/index.js

```diff
@@ -0,0 +1,29 @@
+const got = require('@/utils/got');
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const { id } = ctx.params;
+    const limit = parseInt(ctx.query.limit) || 20;
```

Comment:
```suggestion
    const limit = parseInt(ctx.query.limit) ?? 20;
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator

Reply:
Hi Tony, seems using `??` won't pass Deep Scan. For`parseInt()` will return either `number` or `NaN`, which would never return right-hand value when they are used as left-hand value.
```
NaN ?? 1 = NaN
```
Not sure if you have any further consideration.

Reply:
`const limit = ctx.query.limit ? parseInt(ctx.query.limit) : 20`